csrc/kernels.cu

```diff
@@ -3525,17 +3568,19 @@ template <typename T, int THREADS, int BITS> __global__ void kgemm_4bit_inferenc
   __shared__ T quant_map[16];
 	T local_absmax = T(0.0f);
 
-  for(int i = threadIdx.x; i < 16; i++)
-    quant_map[i] = T(datatype[i]);
+  if (threadIdx.x < 16)
+    quant_map[threadIdx.x] = T(__ldg(&datatype[threadIdx.x]));
+  //for(int i = threadIdx.x; i < 16; i++)
+    //quant_map[i] = T(__ldg(&datatype[i]));
```

Comment:
Not int8 but another small 4bit change that wanted to sneak its way in. @TimDettmers I'm just looking for a sanity check here that this makes sense.